iis-workflows/iis-workflows-core-examples/src/test/java/eu/dnetlib/iis/core/examples/spark/SparkAvroClonerWorkflowTest.java

```diff
@@ -0,0 +1,55 @@
+package eu.dnetlib.iis.core.examples.spark;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.apache.avro.util.Utf8;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import eu.dnetlib.iis.IntegrationTest;
+import eu.dnetlib.iis.core.AbstractOozieWorkflowTestCase;
+import eu.dnetlib.iis.core.OozieWorkflowTestConfiguration;
+import eu.dnetlib.iis.core.WorkflowTestResult;
+import eu.dnetlib.iis.core.examples.schemas.documentandauthor.Person;
+
+/**
+ * @author Łukasz Dumiszewski
+ */
+@Category(IntegrationTest.class)
+public class SparkAvroClonerWorkflowTest extends AbstractOozieWorkflowTestCase {
+
+        private static Logger log = LoggerFactory.getLogger(SparkAvroClonerWorkflowTest.class);
+    
+        @Test
+        public void sparkAvroCloner() throws Exception{
+            
+            
+            // given
+            
+            OozieWorkflowTestConfiguration conf = new OozieWorkflowTestConfiguration();
+            
+            conf.addExpectedOutputAvroDataStore("simple_java_cloner/person/content.avro");
```

Comment:
Did you try with `simple_java_cloner/person` (without `content.avro`)?
I think that providing directory only is little more readable. In workflow.xml you provide only directory. Information that file will be named `content.avro` is hidden deep in `DataStore` class.